docs/fundamentals/code-analysis/overview.md

```diff
@@ -103,30 +105,43 @@ Starting .NET 5.0 RC2, you can enable code style analysis on build, both at the
 Steps to enable code style analysis on build:
 
 1. Set the MSBuild property [EnforceCodeStyleInBuild](../../core/project-sdk/msbuild-props.md#enforcecodestyleinbuild) to `true`.
-2. [Configure](configure-code-analysis-rules.md) each "IDE" code style rule that you wish to run on build as a warning or an error. For example:
+
+1. In an *.editorconfig* file, [configure](configuration-options.md) each "IDE" code style rule that you wish to run on build as a warning or an error. For example:
```

Comment:
Should this be `2.`?

Reply:
It's fine either as a 1 or 2. It will get rendered as a sequential numbered list either way, as long as they start at the same indentation.

![image](https://user-images.githubusercontent.com/24882762/94216833-4a65f700-fe95-11ea-8d49-a78e5a2ebf0e.png)